tensorflow/core/ops/array_ops.cc

```diff
@@ -410,6 +410,28 @@ axis: Dimension along which to unpack.  Negative values wrap around, so the
 output: The list of tensors unpacked from `value`.
 )doc");
 
+REGISTER_OP("UnravelIndex")
+    .Input("indices: Tidx")
+    .Input("dims: Tidx")
+    .Output("output: Tidx")
+    .Attr("Tidx: {int32, int64} = DT_INT32")
+    .SetShapeFn([](InferenceContext* c) { return Status::OK(); })
+    .Doc(R"doc(
+Converts a flat index or array of flat indices into a tuple of coordinate
+arrays.
```

Comment:
Ops need a 1-line <72 character summary line to become the first line of the Python doc string. Either shorten this or split it into two pieces with a newline between.

Reply:
Thanks @josh11b. The doc string has been updated.